src/main/java/in/muthukumari/controller/BankDetailsController.java

```diff
@@ -38,18 +29,13 @@ public BankDetailsController() {
 	 * @param bankName
 	 * @param branchName
 	 * @return
-	 * @throws InvalidException
-	 * @throws ClassNotFoundException
-	 * @throws SQLException
-	 * @throws DBException 
 	 */
 	public String getIfscCode(String bankName, String branchName)
-			throws InvalidException {
-		
+			 {
 		String ifscCode = null;
 		try {
 			ifscCode = BankDetailService.getIfscCode(bankName, branchName);
-		} catch (ClassNotFoundException | SQLException e) {
```

Comment:
add e.printstacktrace

Reply:
e.printstacktrace shows the issue in sonar cloud. so I didn't use this.